Sources/PointFree/Blog/BlogPosts/BlogPost0001_Announcement.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+let post0001_welcome = BlogPost(
+  blurb: """
+""",
+  contentBlocks: [
+    .init(
+      content: """
+      It's hard to believe it's already been X (TODO) months since we launched Point-Free, but
```

Comment:
just what I came up for a first post off the top of my head. think about what other things you might wanna say here.